modules/mauth-test-utils/src/main/java/com/mdsol/mauth/test/utils/model/SigningAuthentication.java

```diff
@@ -0,0 +1,37 @@
+package com.mdsol.mauth.test.utils.model;
+
+public class SigningAuthentication implements TestCase {
+  String name;
+  CaseType type;
```

Comment:
This field is unnecessary I think. Everything can be in `getType`  impl

Reply:
Updated as suggested in https://github.com/mdsol/mauth-jvm-clients/pull/133/commits/a33ca9d7f45ba653dacdc14bbcef7d90d4c2a853